wp-content/themes/pub/wporg-learn-2020/front-page.php

```diff
@@ -38,6 +38,10 @@
 				<a class="section-heading_link" href="/courses/"><span aria-hidden="true"><?php esc_html_e( 'View All Courses', 'wporg-learn' ); ?></span><span class="screen-reader-text"><?php esc_html_e( 'View All Courses', 'wporg-learn' ); ?></span></a>
 			</div>
 
+			<p>
+				<em><?php esc_html_e( 'Multimedia courses setting you up to become a better WordPress developer, designer, user, and contributor.', 'wporg-learn' ); ?></em>
+			</p>
```

Comment:
I think this needs some space above it. There are existing styles; if you add this class:
```suggestion
			<p class="section-heading_description">
				<em><?php esc_html_e( 'Multimedia courses setting you up to become a better WordPress developer, designer, user, and contributor.', 'wporg-learn' ); ?></em>
			</p>
```

and move it within the `.section-heading` above it should update to look like this

![Screen Shot 2022-12-05 at 12 28 47 PM](https://user-images.githubusercontent.com/1017872/205522356-2005142e-16e8-4f7b-b8d2-310e1fd6bba3.jpg)

![Screen Shot 2022-12-05 at 12 28 20 PM](https://user-images.githubusercontent.com/1017872/205522370-9b358920-06ea-4b65-8969-58265a49d8df.jpg)